ownCloudTests/File List/CreateFolderTests.swift

```diff
@@ -0,0 +1,319 @@
+//
+//  CreateFolderTests.swift
+//  ownCloudTests
+//
+//  Created by Javier Gonzalez on 08/01/2019.
+//  Copyright © 2019 ownCloud GmbH. All rights reserved.
+//
+
+import XCTest
+import EarlGrey
+import ownCloudSDK
+import ownCloudMocking
+
+@testable import ownCloud
+
+class CreateFolderTests: XCTestCase {
+
+	let hostSimulator: OCHostSimulator = OCHostSimulator()
+
+	override func setUp() {
+		super.setUp()
+	}
+
+	override func tearDown() {
+		super.tearDown()
+		OCMockManager.shared.removeAllMockingBlocks()
+	}
+
+	public typealias OCMRequestCoreForBookmarkCompletionHandler = @convention(block)
+		(_ core: OCCore, _ error: NSError?) -> Void
+
+	public typealias OCMRequestCoreForBookmark = @convention(block)
+		(_ bookmark: OCBookmark, _ completionHandler: OCMRequestCoreForBookmarkCompletionHandler) -> OCCore
+
+	public typealias OCMRequestChangeSetWithFlags = @convention(block)
+		(_ flags: OCQueryChangeSetRequestFlag, _ completionHandler: OCQueryChangeSetRequestCompletionHandler) -> Void
+
+	/*
+	* PASSED if: Create Folder view is shown
+	*/
+	func testShowCreateFolder() {
+
+		if let bookmark: OCBookmark = UtilsTests.getBookmark() {
+			//Mocks
+			self.mockOCoreForBookmark(mockBookmark: bookmark)
+			self.mockQueryPropfindResults(resourceName: "PropfindResponse", basePath: "/remote.php/dav/files/admin", state: .contentsFromCache)
+			self.showFileList(bookmark: bookmark)
+
+			//Actions
+			EarlGrey.select(elementWithMatcher: grey_accessibilityID("sort-bar.leftButton")).perform(grey_tap())
+
+			//Assets
```

Comment:
Assets -> Asserts

Reply:
Changed here and in other tests that were wrong 👍